### PR TITLE
Add OFFIS as partner to README

### LINKED ISSUE #490: Add research institute OFFIS as development partner
My new employer is [OFFIS](https://www.offis.de/), so the institute should be added as a partner here:

- [ ] Add logo to README
- [ ] Add copyright to README
- [ ] Maybe add OFFIS as partner of OEP

### PR DESCRIPTION
**Pull request by FlorianK13**
### PR-Assignee
- [x] 🐙 Follow the workflow in [CONTRIBUTING.md](https://github.com/OpenEnergyPlatform/open-MaStR/blob/production/CONTRIBUTING.md)
- [x] 📝 Update the [CHANGELOG.md](https://github.com/OpenEnergyPlatform/open-MaStR/blob/production/CHANGELOG.md)
- [ ] 📙 Update the documentation

### Reviewer
- [x] 🐙 Follow the [Reviewer Guidelines](https://github.com/OpenEnergyPlatform/open-MaStR/blob/production/CONTRIBUTING.md#40-let-someone-else-review-your-pr)
- [x] 🐙 Provided feedback and show sufficient appreciation for the work done

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -1,5 +1,5 @@
 
-.. image:: https://user-images.githubusercontent.com/14353512/199113556-4b53660f-c628-4138-8d01-3719595ecda1.png
+.. image:: https://private-user-images.githubusercontent.com/74312290/311242144-1992d975-c410-4cb9-8a05-117731d37084.svg?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MDk5MDM2MDMsIm5iZiI6MTcwOTkwMzMwMywicGF0aCI6Ii83NDMxMjI5MC8zMTEyNDIxNDQtMTk5MmQ5NzUtYzQxMC00Y2I5LThhMDUtMTE3NzMxZDM3MDg0LnN2Zz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDAzMDglMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwMzA4VDEzMDgyM1omWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTA0MmE0YTE2ZmNiNGYzYjVhNDMwZTljMmU3NWM1YTkxMDM5MDJiYjA2YTUwMDE4ZDJjNDA5ZDc5NzgwYTIyNmUmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.jtYbCYD9mbLe-U76BKpjY7p6jXKkAbvEdwp0oocx1Tw
```
Comment (FlorianK13) [inline README.rst]: private User image does not work I think, I need to upload it properly.

File: README.rst [hunk 2]
```diff
@@ -1,5 +1,5 @@
 
-.. image:: https://user-images.githubusercontent.com/14353512/199113556-4b53660f-c628-4138-8d01-3719595ecda1.png
+.. image:: https://raw.githubusercontent.com/OpenEnergyPlatform/open-MaStR/feature-490-add-offis-as-partner/docs/images/README_HeaderThreePartners.svg
```
Comment (nesnoj) [inline README.rst]: As soon as the branch is deleted, this image won't be available no more, will it?

File: README.rst [hunk 3]
```diff
@@ -59,8 +59,6 @@ Documentation
 | Find the `documentation <https://open-mastr.readthedocs.io/en/latest/>`_ hosted on ReadTheDocs.
 
 | The original API documentation can be found on the `Webhilfe des Marktstammdatenregisters <https://www.marktstammdatenregister.de/MaStRHilfe/subpages/webdienst.html>`_.
-| If you are interested in browsing the MaStR online, check out the privately hosted `Marktstammdatenregister.dev <https://marktstammdatenregister.dev/>`_.
-| Also see the `bundesAPI/Marktstammdaten-API <https://github.com/bundesAPI/marktstammdaten-api>`_ for another implementation.
```
Comment (nesnoj) [inline README.rst:63]: Why did you remove these links? They're are not essential but could be helpful. Maybe we could move them to the docs?  To my mind they do not really fit into the current structure but what so you think about a new section after [Development](https://open-mastr.readthedocs.io/en/latest/development/contributing_mirror/) with helpful links in general?

Comment (FlorianK13) [reply README.rst:63]: Yes, you are right. I remember removing them because we got a bug report for marktstammdatenregister.dev once, so we might rephrase it that it is clear that this is not from us.

File: README.rst (hunk 2 above)
Comment (FlorianK13) [reply README.rst]: Do you know how to get this user-images link? For the original one I imagine it went this way:
* Having a README in .md format
* Drag and drop the image in the github editor
* Convert .md to .rst

Now that we have .rst we cannot drag and drop the image to the README anymore.

One solution is to copy the image to the issue #490 and use the generated link from there, hoping that issue images will not be deleted sometimes.

Comment (nesnoj) [reply README.rst]: That could work, however the subdomain is "private-user-images" so I'm not sure if this will work. Also, as you mentioned, it will be lost once the issue is removed.

Therefore, I propose a cleaner solution: I think the logo belongs to the repo data. So I would commit the image to `docs/images/` and add the relative path to the readme like this:
```
.. image:: docs/images/LOGONAME.svg
```

I tested with an existing image in branch [test-rst-local-image](https://github.com/OpenEnergyPlatform/open-MaStR/tree/test-rst-local-image), works out.

What do you think?

File: README.rst (hunk 3 above)
Comment (nesnoj) [reply README.rst:63]: Keeping and rephrasing sounds good to me :+1:

File: README.rst (hunk 2 above)
Comment (FlorianK13) [reply README.rst]: Yes this looks like a very good solution 👍

Comment (FlorianK13) [reply README.rst]: I hope it will also work for the pypi page of open-mastr

File: README.rst (hunk 3 above)
Comment (FlorianK13) [reply README.rst:63]: I just checked and https://marktstammdatenregister.dev seems to be down (not the landing page, but the actual datasette pages where you can click through the dataset). Should we still mention it @nesnoj?

Comment (nesnoj) [reply README.rst:63]: Yes, but I'd use the github orga: https://github.com/marktstammdatenregister-dev

(BTW: They have [Zenodo releases of the data](https://zenodo.org/records/10427795) but it no longer seems to be maintained - time for me to boost #559 ;)

File: README.rst (hunk 2 above)
Comment (nesnoj) [reply README.rst]: > I hope it will also work for the pypi page of open-mastr

The image is located at a separate location (pypi-camo.freetls.fastly.net...) so probably copied on release. Not sure but I guess it will work out..

**Review by nesnoj — APPROVED** [review]
LGTM!
